--- rip.ori	2020-08-04 11:37:56.241011787 +0200
+++ rip.pl	2020-07-30 17:29:30.779142046 +0200
@@ -1,4 +1,4 @@
-#! c:\perl\bin\perl.exe
+#!/usr/bin/perl
 #-------------------------------------------------------------------------
 # Rip - RegRipper, CLI version
 # Use this utility to run a plugins file or a single plugin against a Reg
@@ -63,11 +63,11 @@
 # Suggested addition by Hal Pomeranz for compatibility with Linux
 #push(@INC,$str);
 # code updated 20190318
-my $plugindir;
-($^O eq "MSWin32") ? ($plugindir = $str."plugins/")
-                   : ($plugindir = File::Spec->catfile("plugins"));
+#my $plugindir;
+#($^O eq "MSWin32") ? ($plugindir = $str."plugins/")
+#                   : ($plugindir = File::Spec->catfile("plugins"));
 #my $plugindir = $str."plugins/";
-#my $plugindir = File::Spec->catfile("plugins");
+my $plugindir = "/opt/regripper/plugins/";
 #print "Plugins Dir = ".$plugindir."\n";
 # End code update
 my $VERSION = "3\.0";
@@ -557,4 +557,4 @@
 	my ($sec,$min,$hour,$mday,$mon,$year) = gmtime($epoch);
 	my $fmt = sprintf("%04d-%02d-%02d %02d:%02d:%02d",(1900 + $year),($mon + 1),$mday,$hour,$min,$sec);
 	return $fmt;
-}
\ No newline at end of file
+}
